pkgs/development/ocaml-modules/ffmpeg/ffmpeg-swscale.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildDunePackage, dune-configurator, pkg-config, fetchFromGitHub
+, ffmpeg-base ? import ./base.nix { inherit lib fetchFromGitHub; }
```

Comment:
```suggestion
, ffmpeg-base ? callPackage ./base.nix { }
```
Also if ffmpeg-base is provided all the time we can remove the `? ...` part.

Reply:
ffmpeg-base isn't explictly provided all the time, but like, I guess it could be? Feel free to expand on what you mean